librz/core/cmd_type.c

```diff
@@ -1735,3 +1822,162 @@ RZ_IPI int rz_cmd_type(void *data, const char *input) {
 	}
 	return true;
 }
+
+RZ_IPI RzCmdStatus rz_type_cc_list_handler(RzCore *core, int argc, const char **argv, RzOutputMode mode) {
+	const char *cc = argc > 1 ? argv[1] : NULL;
+	if (cc) {
+		types_cc_print(core, cc);
+	} else {
+		types_cc_print_all(core, mode);
+	}
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_type_cc_del_handler(RzCore *core, int argc, const char **argv) {
+	const char *cc = argc > 1 ? argv[1] : NULL;
+	if (!cc) {
+		eprintf("Specify the calling convention");
+		return RZ_CMD_STATUS_ERROR;
```

Comment:
WRONG_ARGS is better

Reply:
Also, why are you checking? In general, the parser will automatically check for the right number of arguments